Free trial modal

berntgl opened the pull request:
Added modal to "Get started for free" button on landing page.

Note: we'll have to tweak the js code a bit if we want to add more modals in the future. Right now it's only set up for the single modal.

yakkomajuri commented:
<img width="1410" alt="Screenshot 2020-09-24 at 16 33 32" src="https://user-images.githubusercontent.com/38760734/94173626-b9d2eb00-fe83-11ea-8aa0-57a05179a16f.png">

berntgl commented:
Oof that is rough, let me see what's up

Twixes commented:
A few small notes:
- I feel like the "SaaS/Cloud" and "Self Deploy" headings should not change color on option hover.
- The modal should have a shadow instead of a border, and be more rounded (as rounded as its options).
- I'd change the copy a bit to:

| Cloud | Self-Deployed |
|:---|:---|
| PostHog as a Service | Host Your Own PostHog |
| The quickest way to get valuable insights into your product: try PostHog out right away by signing up for PostHog Cloud. Let us handle all the maintenance and scalability for you. | Easily set up a PostHog instance on your own infrastructure of choice and retain absolute power over your deployment and your data. Full control. |

berntgl commented:
Thanks for the feedback @Twixes ! I've changed the border of the modal to a shadow. I'm not sure what you mean by "the "SaaS/Cloud" and "Self Deploy" headings should not change color on option hover." When hovering over the cards, it should just give a blue border. Also I like your descriptions, but I'm keen to hear what others think before I add that. 

Also added: 
- Styling for the mobile version

jamesefhawkins commented:
I aligned the wording with our (new) pricing page better as the main change -just waiting for build to double check before merging.

Thanks all!!
